chef-config/lib/chef-config/workstation_config_loader.rb

```diff
@@ -17,6 +17,7 @@
 #
 
 require "chef-utils" unless defined?(ChefUtils::CANARY)
+require "etc" unless defined?(Etc)
```

Comment:
Just FTR for everyone else, we use `Etc` on line 226 in this file, so this is obviously correct.